packages/doc-site/src/components/Dial.md

```diff
@@ -0,0 +1,150 @@
+# Overview
+
+```jsx
+import { COMPARISON_OPERATOR, StatusIcon, StreamType, DataType } from '@synchro-charts/core';
+<div style={{ width: '100%', height: '300px' }}>
+  <Dial
+    widgetId="test-widget"
+    dataStream={{
+      id: 'car-speed-alarm',
+      name: 'Wind temperature',
+      data: [
+        {
+          x: new Date(2001, 0, 0).getTime(),
+          y: 900,
+        },
+      ],
+      resolution: 0,
+      dataType: DataType.NUMBER,
+    }}
+    viewport={{ yMin: 0, yMax: 2000, duration: 1000 }}
+  />
+</div>;
+```
+
+```jsx
+import { COMPARISON_OPERATOR, StatusIcon, StreamType, DataType } from '@synchro-charts/core';
+<div style={{ width: '100%', height: '300px' }}>
+  <Dial
+    widgetId="test-widget"
+    dataStream={{
+      id: 'car-speed-alarm',
+      name: 'Wind temperature',
+      data: [
+        {
+          x: new Date(2001, 0, 0).getTime(),
+          y: 700,
+        },
+      ],
+      unit: 'rpm',
+      resolution: 0,
+      dataType: DataType.NUMBER,
+    }}
+    viewport={{ yMin: 0, yMax: 2000, duration: 1000 }}
+  />
+</div>;
+```
+
+```jsx
+import { COMPARISON_OPERATOR, StatusIcon, StreamType, DataType } from '@synchro-charts/core';
+<div style={{ width: '100%', height: '300px' }}>
+  <Dial
+    widgetId="test-widget"
+    dataStream={{
+      id: 'car-speed-alarm',
+      name: 'Wind temperature',
+      data: [
+        {
+          x: new Date(2001, 0, 0).getTime(),
+          y: 700.354,
+        },
+      ],
+      unit: 'rpm',
+      resolution: 0,
+      dataType: DataType.NUMBER,
+    }}
+    viewport={{ yMin: 0, yMax: 2000, duration: 1000 }}
+  />
+</div>;
+```
+
+### Dial with alarms
+
+You can show the current latest alarm status on the legend through setting the `icon` property for the threshold.
```

Comment:
this is copy-pasta. Please re-read through this and edit as appropriate.

Reply:
Done